src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -350,6 +363,34 @@ public void postPatronAccountInstanceHoldByIdAndInstanceId(String id,
     }
   }
 
+  @Override
+  public void getPatronAccountInstanceAllowedServicePointsByIdAndInstanceId(String requesterId,
+    String instanceId, Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    var httpClient = HttpClientFactory.getHttpClient(vertxContext.owner());
+    String url = format(CIRCULATION_REQUESTS_ALLOWED_SERVICE_POINTS, requesterId, instanceId);
+
+    try {
+      httpClient.get(url, okapiHeaders)
```

Comment:
can we consider including httpClient.get into future to avoid try/catch block?

Reply:
done